patches/lua-v5.4.4.patch

```diff
@@ -1,3 +1,25 @@
+diff --git a/lapi.c b/lapi.c
+index 332e97d1..deb638a4 100644
+--- a/lapi.c
++++ b/lapi.c
+@@ -534,6 +534,17 @@ LUA_API const char *lua_pushlstring (lua_State *L, const char *s, size_t len) {
+   return getstr(ts);
+ }
+ 
++LUA_API const char *lua_pushlongstring (lua_State *L, const char *s, size_t len) {
++  TString *ts;
++  lua_lock(L);
++  ts = luaS_createlngstrobj(L, len);
```

Comment:
This looks good, I would just test manually that the flow works ok with empty string

Reply:
Ah? how is this working? shouldnt you remove lines 18 and 19 with this change of if else?

Reply:
you are right, sorry. Changed